specification/web/resource-manager/Microsoft.Web/stable/2021-03-01/Certificates.json

```diff
@@ -0,0 +1,710 @@
+        "description": "Description for Delete a certificate.",
+        "operationId": "Certificates_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the certificate.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully deleted certificate."
+          },
+          "204": {
+            "description": "Certificate does not exist."
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete Certificate": {
+            "$ref": "./examples/DeleteCertificate.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Certificates"
+        ],
+        "summary": "Create or update a certificate.",
+        "description": "Description for Create or update a certificate.",
+        "operationId": "Certificates_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the certificate.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "certificateEnvelope",
+            "in": "body",
+            "description": "Details of certificate, if it exists already.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CertificatePatchResource"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/Certificate"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Patch Certificate": {
+            "$ref": "./examples/PatchCertificate.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/certificates/{name}/hostNameBindings": {
```

Comment:
>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/certificates/{name}/hostNameBindings" [](http://example.com/codeflow?start=5&length=128)

A point GET for hostNameBindings also needs to be added. Otherwise, Azure policy won't support this resource type.

[https://armwiki.azurewebsites.net/api_contracts/guidelines/policy.html#plcy008-support-a-collection-get-and-individual-get-for-child-resource-types-even-singletons](https://armwiki.azurewebsites.net/api_contracts/guidelines/policy.html#plcy008-support-a-collection-get-and-individual-get-for-child-resource-types-even-singletons)

Reply:
A task isn't enough.  This needs to be implemented before adding this API.  sites/hostnamebindings (and sites/slots/hostnamebindings) already supports this and it is a standard contract.  Please remove this API from the spec if you aren't able to implement this for this checkin

Reply:
API has been removed. @pilor Let me know if there are any further changes required.